app/views/tag_groups/_form.html.erb

```diff
@@ -0,0 +1,26 @@
+<% modal ||= false %>
```

Comment:
And when will this not be passed to justify the use of `||=`? If you don't pass a local variable to a partial it'll simply break so this `||=` here won't work.